S2VX.Game/Story/Note/Approaches.cs

```diff
@@ -47,9 +49,74 @@ public class Approaches : CompositeDrawable {
         [BackgroundDependencyLoader]
         private void Load() => RelativeSizeAxes = Axes.Both;
 
-        protected override void Update() {
+        private void UpdateApproach(Approach approach, double approachTime, List<RelativeBox> approachLines) {
             var notes = Story.Notes;
-            Alpha = notes.Alpha;
+            var camera = Story.Camera;
+            var approaches = Story.Approaches;
+
+            var time = Time.Current;
+            var endFadeOut = approachTime + notes.FadeOutTime;
+
+            if (time >= endFadeOut) {
+                approachLines.ForEach(l => l.Alpha = 0); // Lines vs ReleaseLines
```

Comment:
I see that this function replaces and handles both lines from Approach and HoldApproach but what does this comment mean / why is it useful?

Reply:
Oh I accidentally left that in there, that was a comment for myself. Removed